fontforge/splinefit.c

```diff
@@ -0,0 +1,1067 @@
+/* -*- coding: utf-8 -*- */
```

Comment:
For my understanding, is there any actual new code in this file, or is it just a straight copy out? If it's not a straight copy, what's new?

Reply:
Ah yes -- sorry. There could be some tiny changes above `_ApproximateSplineSetFromGen()` but basically everything above that function is a copy and those last two functions are new.

Reply:
Oh, I guess I should take that back. 

There was an old structure called "TPoint" that had x, y, and t values and was used by a number of functions. I replaced that with a FitPoint structure that puts x and y into a BasePoint and adds a slope. Most functions that use TPoints were rewritten to use FitPoints while ignoring the slope. So while this winds up being mostly syntactic I can't honestly call it "tiny".

Reply:
The easiest method of review for that upper material is probably to make a local git clone and use blame -C on the file. I just tried this and it's pretty tractable.